python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py

```diff
@@ -106,7 +104,7 @@ def add_field_surveys(
                     observation_reason=observation_reason,
                     scheduler_note=None,
                     readtime=2.4,
```

Comment:
We can actually remove the `readtime` and `band_change_time` overrides now and go with the default model parameters since we are finally using the LSSTCam :)